### DISCUSSION TITLE
TS0601 programming_mode proper format? (No converter available for...)

### DISCUSSION THREAD

**Discussion by JakoobCZ** [Koenkk/zigbee2mqtt#16532] [Devices]
Hi,
I have reached a dead end and any help would be appreciated. I'm running TUYA HY368 (TS0601) radiator valves and I tried to schedule the change in temperature using the web interface of HA, specifically the "programming_mode" in "Exposes", but I can't find nor guess the proper format.

**If select 5+2 and then input:**
06:00/21°C 08:00/15°C 11:30/15°C 12:30/15°C 18:00/21°C 22:00/18°C //for workdays_schedule
06:00/18°C 08:00/21°C 11:30/21°C 12:30/21°C 18:00/21°C 22:00/18°C //for holidays_schedule
![image](https://user-images.githubusercontent.com/31956337/216790546-4a5955aa-35ac-4f97-9e3f-405d5ce5497f.png)

**I get a red error message saying:**
"No converter available for 'programming_mode' ..." 
![image](https://user-images.githubusercontent.com/31956337/216790525-82042cb6-ca48-49d0-a86c-b275b0eea58d.png)

I tried to search for documentation or a guide, but so far no luck.

**System info:**

- Home Assistant 2023.1.4
- Supervisor 2023.01.1
- Operating System 9.4
- Zigbee2MQTT 1.30.0-1
- RPi3 + Sonoff Dongle-E

Thanks for the help :) 

_I tried installing the NTP server as suggested [here](https://github.com/Koenkk/zigbee2mqtt/discussions/11103), but didn't help._

**Comment by leblitzdick** [comment]
I have the same problem as you, it is related to this:

https://github.com/Koenkk/zigbee2mqtt/issues/16161

**Comment by ottoketney** [comment]
I have same behavour
